eli5/formatters/utils.py

```diff
@@ -0,0 +1,35 @@
+import re
+
+
+def replace_starting_trailing_spaces(s, replacer):
```

Comment:
Spaces within features also look fine, so I added highlighting of all spaces and the title attribute:
![2016-10-24 17 48 12](https://cloud.githubusercontent.com/assets/424613/19650486/3250e1ca-9a12-11e6-9bcd-b337e724decd.png)